Examples/SnapinsSDKKotlinExample/app/src/javaImplDebug/java/com/salesforce/snapinssdkexample/ServiceSDKApplication.java

```diff
@@ -0,0 +1,29 @@
+package com.salesforce.snapinssdkexample;
+
+import android.app.Activity;
+import android.app.Application;
+
+import com.salesforce.android.chat.core.internal.liveagent.ChatStateListener;
+import com.salesforce.androidsdk.app.SalesforceSDKManager;
+import com.salesforce.snapinssdkexample.activities.MainActivity;
+
+public class ServiceSDKApplication extends Application {
+    private ChatSessionListener chatSessionListener;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        initializeSalesforceSDKManager(MainActivity.class);
```

Comment:
This line could use a comment something like...
`// Initializes the Salesforce Mobile SDK so that we can use it for authentication`
I'm not sure if that's accurate, but just using it as an example

Reply:
done